modules/ROOT/pages/dataweave-troubleshoot.adoc

```diff
@@ -264,5 +264,53 @@ read(payload.^raw replace "\uFFFF" with "NonChar")
 
 == Low Performance Issues
 
+=== application/dw format
 Using `output application/dw` format can impact the performance of transformations. This format is intended to help you debug the results of DataWeave transformations.
 It is significantly slower than other formats, so avoid using this format in production applications.
+
+=== multipart/form-data content parsing
+When working with `multipart/form-data` inputs, trying to access the content of a large part could cause performance degradation. This happens because when accessing the content of a part, the interpreter will try to parse it for further querying and analysis which can take time specially for high memory formats like `application/xslx`. 
```

Comment:
```suggestion
When working with `multipart/form-data` inputs, trying to access the content of a large part could cause performance degradation. This happens because when accessing the content of a part, the interpreter will try to parse it for further querying and analysis which can take time specially for high memory formats like `application/xlsx`. 
```